internal/evaluator/conftest_evaluator_test.go

```diff
@@ -1282,6 +1282,33 @@ func TestConftestEvaluatorIncludeExclude(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "warning for missing includes",
+			results: []Outcome{
+				{
+					Failures: []Result{
+						{Metadata: map[string]any{"code": "lunch.spam"}},
+						{Metadata: map[string]any{"code": "breakfast.spam", "term": []any{"bacon", "nocab"}}},
+					},
+				},
+			},
+			config: &ecc.EnterpriseContractPolicyConfiguration{Include: []string{"breakfast:eggs", "lunch"}},
+			want: []Outcome{
+				{
+					Skipped:    []Result{},
+					Warnings:   []Result{},
+					Exceptions: []Result{},
+					Failures: []Result{
+						{Metadata: map[string]any{"code": "lunch.spam"}},
+					},
+				},
+				{
+					Warnings: []Result{
+						{Message: "Include criterion 'breakfast:eggs' doesn't match any policy rule"},
+					},
```

Comment:
Looks like we're producing a list of length 2 in `[]Outcome`, but all the other tests here have just single outcome in `[]Outcome`. Is that okay? I was wondering if the `Result` here could or should be merged into the `Warnings` field on line 1299.

(I'm not sure though. I might try running this with a real example to see what output gets produced.)

Reply:
Based on the acceptance test snapshot, seems like it all gets collated together, so perhaps this is fine.